### DISCUSSION TITLE
Flutterfire won't configure

### DISCUSSION THREAD

**Discussion by D4N4PP** [firebase/flutterfire#11704] [General] [answered]
I am trying to create a firebase project. But when I try to create one using flutterfire, this is what happens:

`✔ Enter a project id for your new Firebase project (e.g. my-cool-project) · fluttercourse-yung

i New Firebase project fluttercourse-yung created successfully.

FirebaseCommandException: An error occured on the Firebase CLI when attempting to run a command.

COMMAND: firebase projects:create fluttercourse-yung --json

ERROR: Failed to create project. See firebase-debug.log for more info.`

**Comment by pelumibabs** [comment, marked as answer]
Enter your project name like this

> my-project-001

"-" is important
